.spelling

```diff
@@ -777,6 +778,7 @@ react-native-bpk-component-progress
 react-native-bpk-component-section-list
 react-native-bpk-component-select
 react-native-bpk-component-spinner
+react-native-bpk-component-split-input
```

Comment:
do we need this? 🤔

Reply:
Adressed in : [6b52512](https://github.com/Skyscanner/backpack/pull/2472/commits/6b5251262b5790f7faf197c67ff4ee2381cf2b92)

Reply:
I thought I had already committed that, updated now :)